MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+                            </ViewCell.View>
+                        </ViewCell>
+                    </DataTemplate>
+                </listView:SfListView.ItemTemplate>
+            </listView:SfListView>
+        </Grid>
+    </ContentPage.Content>
+</ContentPage>
+
+{% endhighlight %}
+{% endtabs %}
+
+## ListView with Prism Framework
+
+The `SfListView` allows the user to work with prism for MVVM Framework. Steps to be followed:
+
+1. Replace your application to prism application in App.xaml file.
+2. Inherit App.xaml.cs from `Prism` application instead of your application.
+3. Create a prism namespace library reference in xaml file of the `ContentPage`.
+4. Connect view and view model instead of binding context by registering them.
+
+{% tabs %}
+{% highlight c# %}
+public partial class App : PrismApplication
+{
+    public App(IPlatformInitializer initializer = null) : base(initializer) { }
+
+    protected override void OnInitialized()
+    {
+        InitializeComponent();
+
+    }
+
+    protected override void RegisterTypes()
+    {
+        Container.RegisterTypeForNavigation<MainPage, MainPageViewModel>();
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight xaml %}
+<?xml version="1.0" encoding="utf-8" ?>
+<prism:PrismApplication xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                        xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                        xmlns:prism="clr-namespace:Prism.Unity;assembly=Prism.Unity.Forms"
+                        x:Class="ListViewPrism.App">
+</prism:PrismApplication>
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:prism="clr-namespace:Prism.Mvvm;assembly=Prism.Forms"
+             xmlns:local="clr-namespace:ListViewPrism;assembly=ListViewPrism"
+             xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView"
+             x:Class="ListViewPrism.Views.MainPage"
+             Title="MainPage">
+    <StackLayout HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand">
+        <syncfusion:SfListView x:Name="listView" ItemSize="70" ItemSpacing="0,0,5,0"
+                             AutoFitMode="Height"
+                             ItemsSource="{Binding ContactsInfo,}" IsStickyHeader="True" 
+                             AllowSwiping="True" IsStickyGroupHeader="True" GroupHeaderSize="50">
+        </syncfusion:SfListView>
+    </StackLayout>
+</ContentPage>    
+{% endhighlight %}
+{% endtabs %}
+
+## Improving ListView performance
+
+The .NET MAUI ListView (SfListView) has been built from the ground up with an optimized view reuse strategy for the best possible performance on the .NET MAUI platform even when loading large data sets. Following techniques are used to improve performance of the `SfListView`:

```

Comment:
The .NET MAUI ListView (SfListView) has been built from the ground up with an optimized view reuse strategy to provide best possible performance on the .NET MAUI platform, even when loading large data sets. The following techniques are used to improve performance of the `SfListView`:

Reply:
I have modified the line as per the concerns provided